core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -352,7 +356,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
   }
 
   @Test
-  def testConsumeWithNoTopicAccess() {
+  def testConsumeWithoutTopicDescribeAccess() {
```

Comment:
Yes it's the same issue we've discussed above. Until we decide how we want to handle OTOP, we'll have some ugly code.